tests/admin_settings_aws_region_test.php

```diff
@@ -36,6 +36,17 @@
  */
 class admin_settings_aws_region_test extends \advanced_testcase {
 
+    /**
+     * Cleanup after all tests are executed.
+     *
+     * @return void
+     */
+    public static function tearDownAfterClass(): void {
```

Comment:
Is there a reason this is in tearDownAfterClass() rather than just plain Teardown?